ginivision/src/main/java/net/gini/android/vision/camera/CameraFragmentImpl.java

```diff
@@ -324,6 +360,45 @@ public void onClick(final View view) {
                 mFragment.startActivityForResult(fileChooserIntent, REQ_CODE_CHOOSE_FILE);
             }
         });
+        mUploadHintCloseButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View view) {
+                closeUploadHintPopUp();
+            }
+        });
+    }
+
+    private void closeUploadHintPopUp() {
```

Comment:
Awesome idea of using a square view rotated by 45° for the arrow! There is only one problem: the fade in/out animation reveals the covered part of the arrow:
![tip-close-animation](https://user-images.githubusercontent.com/13149498/30813603-2a4c2a66-a20e-11e7-9a30-762aa346f7f4.gif)